pkg/image/apis/image/helper.go

```diff
@@ -494,9 +494,9 @@ func ImageWithMetadata(image *Image) error {
 			image.DockerImageLayers[i].MediaType = layer.MediaType
 		}
 		// reverse order of the layers for v1 (lowest = 0, highest = i)
-		for i, j := 0, len(image.DockerImageLayers)-1; i < j; i, j = i+1, j-1 {
-			image.DockerImageLayers[i], image.DockerImageLayers[j] = image.DockerImageLayers[j], image.DockerImageLayers[i]
-		}
+		// for i, j := 0, len(image.DockerImageLayers)-1; i < j; i, j = i+1, j-1 {
```

Comment:
@smarterclayton https://github.com/docker/distribution/blob/master/docs/spec/manifest-v2-2.md#image-manifest-field-descriptions

* layers array
  The layer list is ordered starting from the base image (opposite order of schema1).